lib/happiness_delivered.rb

```diff
@@ -0,0 +1,111 @@
+# require 'customer_sms'
+require_relative 'sandcream'
+
+class HappinessDelivered
```

Comment:
I think Class names should be slightly more descriptive maybe, I'm not fully sure what to expect of this class (what it does/how to use it) based on the name. This is just your interactive menu class right>